modules/update-preparing-evaluate-apirequestcount.adoc

```diff
@@ -37,7 +37,7 @@ installplans.v1alpha1.operators.coreos.com                     93
 ====
 You can safely ignore the following entries that appear in the results:
 
-* The `system:serviceaccount:kube-system:generic-garbage-collector` user might appear in the results because it walks through all registered APIs searching for resources to remove.
+* The `system:serviceaccount:kube-system:generic-garbage-collector` and the `system:serviceaccount:kube-system:namespace-controller` users might appear in the results because these services invoke all registered APIs when searching for resources to remove.
```

Comment:
These are usernames, and `oc get apirequestcounts` doesn't actually include them.  We should probably grow an example like:

```console
$ oc get apirequestcounts ingresses.v1beta1.networking.k8s.io \
  -o jsonpath='{range .status.currentHour..byUser[*]}{..byVerb[*].verb}{","}{.username}{","}{.userAgent}{"\n"}{end}' \
  | sort -k 2 -t, -u | column -t -s, -NVERBS,USERNAME,USERAGENT
```

pulled from [this KCS][1], or give some alternative pathway to that username information.

[1]: https://access.redhat.com/articles/6955985

Reply:
We do have this exact example elsewhere on the page [1]. Do you think that this note would be more appropriate to have this IMPORTANT message there instead?

[1] https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount-workloads_updating-cluster-prepare

Reply:
(I'm merging this PR now, but can do a followup PR if it makes sense to move it